tests/components/light/test_mqtt.py

```diff
@@ -678,6 +678,37 @@
     assert (255, 128, 63) == state.attributes['rgb_color']
 
 
+async def test_sending_mqtt_brightness_command_with_template(hass, mqtt_mock):
+    """Test the sending of Color Temp command with template."""
```

Comment:
You've left some references to the color temp test here and in the topic names below